lib/netsuite/records/opportunity_item_list.rb

```diff
@@ -0,0 +1,30 @@
+module NetSuite
+  module Records
+    class OpportunityItemList < Support::Sublist
```

Comment:
@jheth `OpportunityItemList` shouldn't contain any logic aside a namespace definition and a `sublist` call. Ex: https://github.com/NetSweet/netsuite/blob/de60d36403587b9d255b9d85ec1d4651af0281cc/lib/netsuite/records/invoice_item_list.rb#L4-L6